README.md

```diff
@@ -125,78 +125,6 @@ Version 1.0.0 of this module introduces a breaking change: adding the `disable-l
 In either case, upgrading to module version `v1.0.0` will trigger a recreation of all node pools in the cluster.
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
please revert documentation changes, each `make genereate` will erase autogenerated documentation, and should be followed by `make generate_docs`

Reply:
done